app/javascript/gobierto_dashboards/webapp/components/DownloadButton.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div
+    v-if="isShowDownloadDatalinkShown"
+    class="dashboards-home-aside--download-open-data"
+  >
+    <i class="fas fa-table"></i>
+    <a :href="dataDownloadEndpoint" target="_blank">
+      {{ labelDownloadData }}
+    </a>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'DownloadButton',
+  data() {
+    return {
+      labelDownloadData: I18n.t('gobierto_dashboards.shared.download_data'),
+    }
+  },
+  props: {
+    dataDownloadEndpoint: {
+      type: String,
+      default: ''
+    }
+  },
+  created(){
+    this.isShowDownloadDatalinkShown = !!this.dataDownloadEndpoint
```

Comment:
When you declare a property with `this`, and then use it in the DOM, it _should_ be declared in the `data` function.
However, in this case, you can save declare a new variable for that and simply:
`v-if="!!this.dataDownloadEndpoint"`